test/databases/couchdb.spec.js

```diff
@@ -0,0 +1,40 @@
+var should = require('should');
+
+var adapterInit = require('../../lib/adapters/couchdb');
+
+var RSVP = require('rsvp');
+var Promise = RSVP.Promise;
+var _ = require('lodash');
+
+RSVP.on('error', function (err) {
+    console.log('rsvp err handler', err);
+    throw err;
+});
+
+describe('CouchDB adapter', function () {
+  var ids;
+
+  describe('Creation', function () {
+    it('should be able to create document with provided id @now', function (done) {
+      
+      adapterInit({
```

Comment:
This initialisation should happen once per full `CouchDB adapter` test suite, so it's better to keep such things in `before` block.